src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -167,7 +162,7 @@ private void recordRateLimiting(
       for (Product product : products) {
         log.debug("Recording rate-limiting", kv("fulfilmentCode", product.getFulfilmentCode()));
         UniquePropertyReferenceNumber uprn =
-            UniquePropertyReferenceNumber.create(caseDetails.getSample().getUprn());
+            UniquePropertyReferenceNumber.create(caseDetails.getSample().get("uprn"));
```

Comment:
We should have a centralised static final def of the sample attribute names

Reply:
Constants for all know attribute names added to CaseUpdate in common.